AUTHORS

```diff
@@ -0,0 +1 @@
+Andrey Cañón <andrey.canon@edunext.co>
```

Comment:
Can you please add an empty line at the end. That would make the the "No line at the end of file" icon go away